qiskit/providers/ibmq/api/clients/websocket.py

```diff
@@ -259,8 +259,6 @@ def get_job_status(
                                 timeout = original_timeout - (time.time() - start_time)
                             else:
                                 response_raw = yield from websocket.recv()
-                        logger.debug('Received message from websocket: %s',
-                                     response_raw)
```

Comment:
I didn't know the API only sent data when there is a change. The reason I got rid of it was because some of the information contained in the message might be "sensitive", such as the `hub`, `group` and `project`. The idea was to prevent these from showing up in the travis runs. Is the information sensitive enough to censor them before logging the response?

Reply:
Similarly, some endpoints would show the device name as well. An example of one is:
```
qiskit.providers.ibmq.session.request:DEBUG:2020-03-11 10:07:07,690: Endpoint: /devices/ibmq_16_melbourne/queue/status. Method: GET. Request Data: {'allow_redirects': True, 'timeout': (5.0, None)}.
```

Reply:
I don't think they're super sensitive, but it doesn't hurt to filter them out.